### ISSUE TITLE
In High-Contrast mode, the Title Bar should change color when the window is inactive

### ISSUE THREAD

**Issue by rudyhuyn** [Microsoft/calculator#441]
**Describe the bug**
As indicated here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar

> Do make is obvious when your window is active or inactive. At a minimum, change the color of the text, icons, and buttons in your title bar.

This is even more true when High-Contrast is activated to improve accessibility.

The Calculator currently uses the same color when the window is active and inactive.

**Steps To Reproduce**
- Activate High-Contrast
- Launch the Calculator
- Switch between applications
--> the Title bar color doesn't change.

**Expected behavior**
- the title bar should change color

**Screenshots**

![image](https://user-images.githubusercontent.com/1226538/55665898-90d81980-57fb-11e9-9599-fbacc1f40bd2.png)

![image](https://user-images.githubusercontent.com/1226538/55665955-79e5f700-57fc-11e9-9c84-e86885fa3b02.png)

**Device and Application Information (please complete the following information):**
 - OS Build: 10.0.18362.0
 - Architecture: X64
 - Application Version: 10.1812.10048.0

**Comment by rudyhuyn** [comment]
I just noticed the titlebar has also an issue with padding on the right:

![image](https://user-images.githubusercontent.com/1226538/55665978-f2e54e80-57fc-11e9-97c9-7c4cca48f61b.png)

**Comment by grochocki** [comment]
Good catch. Outside of HC, we have acrylic/no acrylic, but we should definitely handle HC case as well.

**Comment by rudyhuyn** [comment]
> Good catch. Outside of HC, we have acrylic/no acrylic, but we should definitely handle HC case as well.

The PR I sent only modify the UI when in HC, but we should probably also modify the color of the title when Acrylic is not activated (Battery Saver, Transparency Effects disabled by users in `Settings > Personalization > Colors`, ... ), I can include that in my current PR if you are ok.

_Impossible to see that the Calculator doesn't have the focus:_
![image](https://user-images.githubusercontent.com/1226538/55679087-fe4a7f80-58b9-11e9-93e5-408d618cb0b2.png)

**Comment by grochocki** [comment]
> we should probably also modify the color of the title when Acrylic is disabled... I can include that in my current PR if you are ok.

Good point. Yes, we can cover that as part of this PR.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by rudyhuyn** [comment]
@grochocki: ok, I will try to update the PR on the following days (It's our annual Hackathon this week)

**Comment by grochocki** [comment]
It appears this was fixed as part of #442, but was not caught by automation.
